src/services/data-targets/data-target.service.ts

```diff
@@ -233,12 +234,18 @@ export class DataTargetService {
             (dataTarget as HttpPushDataTarget).timeout = dataTargetDto.timeout;
             (dataTarget as HttpPushDataTarget).authorizationHeader =
                 dataTargetDto.authorizationHeader;
+        } else if (dataTargetDto.type === DataTargetType.Fiware) {
```

Comment:
We should probably just do the cast once, but I agree that this is the way that it is currently done - so this is fine.

Reply:
fixed